tox.ini

```diff
@@ -129,7 +130,7 @@ pip_pre =
 uv_resolution =
     # The oldestdeps factor is intended to be used to install
     # the oldest versions of all dependencies
-    oldestdeps: lowest-direct
+    oldestdeps: lowest
```

Comment:
Here `lowest` is perhaps more self-explanatory than `lowest-direct` ever was, but if you're editing anyway, maybe add ", so we asked uv to install the lowest version" to the comment.